tasks/install/main.yml

```diff
@@ -5,3 +5,23 @@
 
 - include: redhat.yml
   when: ansible_os_family == 'RedHat'
+
+# Reset rights
+- name: set rabbitmq gid
+  group: name=rabbitmq gid={{ rabbitmq_gid }} state=present
+  when: rabbitmq_gid
+
+- name: set rabbitmq uid
+  user: name=rabbitmq uid={{ rabbitmq_uid }} group=rabbitmq state=present
```

Comment:
No because you may not want to change gid but just uid. That's why it's done just before